### PR TITLE
Subgraph: Update event for schedule along with handlers

### PR DESCRIPTION
**Pull request by Evalir**

### CODE REVIEW

File: packages/govern-subgraph/src/GovernQueue.ts [hunk 1]
```diff
@@ -62,8 +62,10 @@ export function handleScheduled(event: ScheduledEvent): void {
   // should be impossible to get at this stage
   container.config = queue.config
   container.queue = queue.id
+  let config = ConfigEntity.load(queue.config)
+  let scheduleDeposit = CollateralEntity.load(config.scheduleDeposit)
```
Comment (Evalir) [inline packages/govern-subgraph/src/GovernQueue.ts]: I was thinking of creating `loadOrCreate` helper here... but these entities relies on values that can't be left `null`. If we want to still create helpers, we should just raise an error.

File: README.md [hunk 2]
```diff
@@ -60,8 +60,8 @@ To use Govern, feel free to deploy your own registries and factories, but using
 
 #### Rinkeby
 
-- 📜 GovernRegistry: [`0x0cd3621EC403F26ad9F79c3d77B1dda1f8474c6f`](https://rinkeby.etherscan.io/address/0x0cd3621EC403F26ad9F79c3d77B1dda1f8474c6f)
-- 🏭 GovernBaseFactory: [`0x0cd3621EC403F26ad9F79c3d77B1dda1f8474c6f`](https://rinkeby.etherscan.io/address/0x0cd3621EC403F26ad9F79c3d77B1dda1f8474c6f)
+- 📜 GovernRegistry: [`0x0cd3621EC403F26ad9F79c3d77B1dda1f8474c6f`](https://rinkeby.etherscan.io/address/0x87eE5EA31dCf1f526f21Bb576131C37890AE65E0)
+- 🏭 GovernBaseFactory: [`0x0cd3621EC403F26ad9F79c3d77B1dda1f8474c6f`](https://rinkeby.etherscan.io/address/0x615e3d83B8e1403c39F98c7066faA1D9bBF9E867)
```
Comment (bpierre) [inline README.md]:
```suggestion
- 📜 GovernRegistry: [`0x87eE5EA31dCf1f526f21Bb576131C37890AE65E0`](https://rinkeby.etherscan.io/address/0x87eE5EA31dCf1f526f21Bb576131C37890AE65E0)
- 🏭 GovernBaseFactory: [`0x615e3d83B8e1403c39F98c7066faA1D9bBF9E867`](https://rinkeby.etherscan.io/address/0x615e3d83B8e1403c39F98c7066faA1D9bBF9E867)
```

File: docs/developers/historical-deployments.md [hunk 3]
```diff
@@ -23,5 +23,5 @@ Log of deployed govern relevant instances and related infrastructure.
 
 ### Contracts
 
-- 📜 GovernRegistry: [`0x0cd3621EC403F26ad9F79c3d77B1dda1f8474c6f`](https://rinkeby.etherscan.io/address/0x0cd3621EC403F26ad9F79c3d77B1dda1f8474c6f)
-- 🏭 GovernBaseFactory: [`0x0cd3621EC403F26ad9F79c3d77B1dda1f8474c6f`](https://rinkeby.etherscan.io/address/0x0cd3621EC403F26ad9F79c3d77B1dda1f8474c6f)
\ No newline at end of file
+- 📜 GovernRegistry: [`0x0cd3621EC403F26ad9F79c3d77B1dda1f8474c6f`](https://rinkeby.etherscan.io/address/0x87eE5EA31dCf1f526f21Bb576131C37890AE65E0)
+- 🏭 GovernBaseFactory: [`0x0cd3621EC403F26ad9F79c3d77B1dda1f8474c6f`](https://rinkeby.etherscan.io/address/0x615e3d83B8e1403c39F98c7066faA1D9bBF9E867)
```
Comment (bpierre) [inline docs/developers/historical-deployments.md]:
```suggestion
- 📜 GovernRegistry: [`0x87eE5EA31dCf1f526f21Bb576131C37890AE65E0`](https://rinkeby.etherscan.io/address/0x87eE5EA31dCf1f526f21Bb576131C37890AE65E0)
- 🏭 GovernBaseFactory: [`0x615e3d83B8e1403c39F98c7066faA1D9bBF9E867`](https://rinkeby.etherscan.io/address/0x615e3d83B8e1403c39F98c7066faA1D9bBF9E867)
```

File: packages/govern-subgraph/src/GovernQueue.ts (hunk 1 above)
Comment (bpierre) [reply packages/govern-subgraph/src/GovernQueue.ts]: Yes throwing an error sounds good :+1:

**Review by bpierre — APPROVED** [review]
LGTM!

File: packages/govern-subgraph/src/utils/ids.ts [hunk 4]
```diff
@@ -7,3 +7,11 @@ export function buildId(event: ethereum.Event): string {
...
   return containerHash + index.toString()
 }
+
+export function buildEventHandlerId(
+  containerHash: string,
+  eventName: string,
+  logIndex: string
+): string {
+  return containerHash + logIndex.toString()
+}
```
Comment (bpierre) [inline packages/govern-subgraph/src/utils/ids.ts:17]: @Evalir Should we just pass the container hash + the event directly as a second parameter, to make the calls shorter?

**Pull request merged**